print_docs.py

```diff
@@ -641,21 +692,25 @@ def write_redirects(loc_map, file_map):
     write_docs_redirect(decl_name, loc_map[decl_name])
     write_src_redirect(decl_name, loc_map[decl_name], file_map)
 
-def copy_css(path, use_symlinks):
+def copy_css_and_js(path, use_symlinks):
   def cp(a, b):
+    # always remove destination, in case we used -l before
+    # and now want to copy the file
+    try:
+      os.remove(b)
+    except FileNotFoundError:
+      pass
     if use_symlinks:
-      try:
-        os.remove(b)
-      except FileNotFoundError:
-        pass
       os.symlink(os.path.relpath(a, os.path.dirname(b)), b)
     else:
       shutil.copyfile(a, b)
 
   cp('style.css', path+'style.css')
   cp('pygments.css', path+'pygments.css')
-  cp('nav.js', path+'nav.js')
+  shutil.copyfile('nav.js', path+'nav.js')
```

Comment:
This really breaks my workflow.  When changing `nav.js`, I don't want to wait for `doc-gen` every time.

Could we instead include the `add_commit.js` file before `nav.js` and use the `commit` variable from there?

Reply:
Done!